src/schema.sql

```diff
@@ -69,28 +83,27 @@ CREATE TABLE geo (
 CREATE INDEX geo_gist ON geo USING GIST(geom);
 CREATE INDEX geo_idx ON geo(id);
 
-DROP TABLE IF EXISTS styles;
+CREATE TABLE geo_history (
+    id          BIGINT NOT NULL,
+    delta       BIGINT NOT NULL REFERENCES deltas(id),
+    key         TEXT UNIQUE,
+    version     BIGINT NOT NULL,
+    geom        GEOMETRY(GEOMETRY, 4326) NOT NULL,
+    props       JSONB NOT NULL,
+    PRIMARY KEY (id, version)
+);
+CREATE INDEX geo_history_gist ON geo_history USING GIST(geom);
+CREATE INDEX geo_history_idx ON geo_history(id);
+CREATE INDEX geo_history_deltax ON geo_history(delta);
+
 CREATE TABLE styles (
     id          BIGSERIAL PRIMARY KEY,
     name        TEXT NOT NULL,
     style       JSONB NOT NULL,
-    uid         BIGINT NOT NULL,
+    uid         BIGINT NOT NULL REFERENCES users(id),
```

Comment:
Changes seems unrelated to the 'history retrieval schema' I think we should move it to another PR.

Reply:
Removed in https://github.com/mapbox/Hecate/pull/206/commits/7493e0efafaeec0af7b70cbab56df3f1b16525d5 -- while these are changes we should make, i'm removing them from the current PR to reduce scope for what will already be a large change.